tests/pr_test.go

```diff
@@ -143,3 +144,49 @@ func TestRunExistingResourcesExample(t *testing.T) {
 		logger.Log(t, "END: Destroy (existing resources)")
 	}
 }
+
+func TestRunStandardSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
```

Comment:
Any specific reason to run test always in us-south?

Reply:
Sure, I kept into consideration that the default region is `us-south` when tried provisioning the `watsonx Assistant` service. 
One other solution can be to assign randomly the available and supported regions but I also followed the other DA code like `postgres` and `Elastic`.

Do you suggest to use some other region instead?

Reply:
If no region restriction, then it should choose random.

Reply:
If we skip `Region`, then dynamic selection of region happens implicitly, however not all regions are supported so made the change for dynamic region selection.
From this [Reference](https://pkg.go.dev/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.42.5/testhelper#TestOptions.Region) it is not clear what all regions will be considered, so added the logic.